roles/base_fedora/tasks/main.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- name: remove cruft packages
+  yum: name={{ fedora_packages_to_remove }} state=absent
+  when: fedora_packages_to_remove
+
+- name: install all the packages
+  yum: name={{ fedora_packages }} state=installed
+  when: fedora_packages
```

Comment:
I know my files don't do this yet, but I want to switch to the full yaml syntax:
```
- name: install all the packages
  yum:
    name: "{{ item }}"
    state: installed
    with_items: fedora_packages
```
That kind of style.